index.html

```diff
@@ -8,16 +8,14 @@
     <title>To Do</title>
 </head>
 <body>
-    
-    <h1>To Do List</h1>
     <div class="content">
-        <input id="inputAddTask" type="text">
-        <button id="buttonAddTask">Add Task</button>
-        <div class="toDoes" id="toDoesContainer">
-
-        </div>
+        <h1>To Do List</h1>
+        <input class="input" type="text">
+        <button class="btn">Add Task</button>
+        <div class="toDos">
+            
+        </div> 
     </div>
-    
     <script src="main.js"></script>
 </body>
 </html>
```

Comment:
As you noticed, there these warnings at the end your files "No newline at end file". Check this for why is that happening: https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

In VSCode you can add .editorconfig file that will automatically fix that for you. 
https://youtu.be/_fcQDS1iTPw

add this editorconfig configuration:
```
root = true

[*]
charset = utf-8
indent_style = space
indent_size = 2
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
```
More on editorconfig here: 
https://editorconfig.org/